docs/liquidators/manual.md

```diff
@@ -0,0 +1,162 @@
+# Manual Liquidations
+
+Running liquidations on Umee requires four main steps:
+
+- Preparing a liquidator account
+- Querying for eligible liquidation targets
+- Executing a liquidation transaction
+- Selling liquidation rewards for a profit
+
+This document contains notes on how to do these manually - a more sophisticated approach would be to automate one or more steps and monitor the results.
+
+## Preparing a Liquidator Account
+
+Any wallet you control can be used as a liquidator, as long as it has a pool of assets and tokens for gas on the relevant chains.
+
+Here is an example setup:
+- `100 AXL-USDC` from axelar collateralized on [app.umee.cc](https://app.umee.cc/) with no active borrows
+- `1 UMEE` on Umee
+- `1 ATOM` on Cosmos Hub
+- `1 AXL` on Axelar
+- `1 OSMO` on Osmosis
+
+The main collateral will allow you to borrow any tokens required during liquidation, and the other tokens are for gas on their native chains.
```

Comment:
> gas on their native chains

This is not required for liquidation. We can create a new section by explaining additional flows, a liquidator can take.